drivers/os/linux/linux.go

```diff
@@ -191,6 +209,32 @@ func (d *driver) nfsMount(device, target string) error {
 	return nil
 }
 
+func (d *driver) objectivefsMount(device, target string) error {
+	command := exec.Command("mount", "-t", "objectivefs", device, target)
+
+	p, err := ioutil.ReadFile(d.config.GetString("linux.volume.objectivefsPassphraseFilePath"))
+	if err != nil {
+		return goof.WithError(fmt.Sprintf("missing license file: set correct linux.volume.objectivefsPassphraseFilePath"), err)
+	}
+
+	passphrase := string(p) + "\n"
+	passphraseInput := strings.NewReader(passphrase)
+	command.Stdin = passphraseInput
+
+	if command.SysProcAttr == nil {
+		command.SysProcAttr = &syscall.SysProcAttr{}
+	}
+	command.SysProcAttr.Setctty = false
+	command.SysProcAttr.Setsid = true
+
```

Comment:
Can you add some comments to the block above so its clear what its doing and why?